package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "group-income",
   "version": "0.0.1",
-  "contractsVersion": "0.1.4",
+  "contractsVersion": "0.1.5",
   "private": true,
   "description": "",
   "scripts": {
     "backend": "npx nodemon --watch backend/ -- --require ./Gruntfile.js --require @babel/register ./backend/index.js",
     "sass-dev": "node-sass --output-style nested --source-map true -w -r -o ./dist/assets/css ./frontend/assets/style",
     "sass-dist": "node-sass --output-style compressed -o ./dist/assets/css ./frontend/assets/style",
-    "cy:open": "cypress open",
+    "cy:open": "CYPRESS_BASE_URL=http://localhost:8000 cypress open",
```

Comment:
The removal of `baseUrl` field in `cypress.json` in [this](https://github.com/okTurtles/group-income/commit/9bf49a43aa9ad111652ef33d1deb010c551b7b25) recent commit has broken `cypress open` command, so added this env variable.

Reply:
Interesting! Thanks for catching that! /cc @snowteamer